test/integration/host_js_test.rb

```diff
@@ -225,7 +225,7 @@ class HostJSTest < IntegrationTestWithJavascript
       page.find('#check_all').click
 
       # Ensure all hosts are checked
-      assert page.find('input.host_select_boxes').checked?
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
```

Comment:
This can also be expressed as `page.has_checked_field?('input:.host_select_boxes')`, the double negative doesn't read very well IMO. 

It also waits, as you can see it's implemented nearly in the same way as `has_no_selector?` http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#assert_no_selector-instance_method